.github/ISSUE_TEMPLATE/tech-debt.yml

```diff
@@ -2,14 +2,6 @@ name: ⚙ Tech Debt
 description: Tech debt is for code improvements that do not change the user-facing behavior (ie, neither adding features nor fixing bugs).
 labels: "T: tech-debt ⚙️"
 body:
```

Comment:
We still have dupes even with this check; I believe our triage process can take care of this going forward

Reply:
Yeah, ☝️ is probably a good argument for removing it from the templates for bugs/feature requests as well... but I'm not touching those until I finish the readme reorg as I plan to cross link a bunch of stuff back to readme sections...